boards/Dell/OptiPlex_7010_9010/results.csv

```diff
@@ -0,0 +1,197 @@
+Test name,Test ID,OptiPlex 7010/9010  v0.1.0-rc1
```

Comment:
Please change the name in the table and file name from `OptiPlex 7010/9010  v0.1.0-rc1` to `OptiPlex 7010 v0.1.0`. We did not test the 9010 model.  The folder name is fine `boards/Dell/OptiPlex_7010_9010`, so let's leave it as is.

Reply:
https://github.com/Dasharo/osfv-results/compare/c79b8fc895dd83df865ee79ce902c7f9e7921281..346e74c0c7b268eea9e7d9d2922ffe8e5a2f3e31#diff-d81e7cc298c4f5085a8bf46384ea55a4109c76a70a9cb924414085e9512aff0cL1

Changed the column name to `Optiplex 7010` and the filename to `v0.1.0-results.csv` instead so it looks like the VP66xx results